.spin/packages.yml

```
@@ -0,0 +1,3 @@
+- g++
+- unzip
+- zip
```

These are also needed for some special scripts, such as updating expectation files.

```suggestion
- zip
- autoconf
- coreutils
- parallel
```